stripe-core/src/main/java/com/stripe/android/core/exception/StripeException.kt

```diff
@@ -63,8 +63,19 @@ abstract class StripeException(
                     message = throwable.message,
                     cause = throwable
                 )
-                else -> GenericStripeException(throwable)
+                else -> GenericStripeException(throwable, analyticsValue = analyticsValueForThrowable(throwable))
             }
         }
+
+        private fun analyticsValueForThrowable(throwable: Throwable): String? {
+            val throwableClass = throwable.javaClass
+            if (throwableClass.name.startsWith("android.")) {
```

Comment:
can you combine these if statements into one statement?

Reply:
Sure.